pkg/registries/ecr/ecr.go

```diff
@@ -159,7 +164,8 @@ func newRegistry(integration *storage.ImageIntegration, disableRepoList bool,
 		return reg, nil
 	}
 
-	client, err := createECRClient(conf)
+	ctx := context.Background()
```

Comment:
Why not passing ctx to this method?

Reply:
The registry factory framework is not context aware. I thought that passing context through would be too big of a refactor for now.